src/components/htmlText/index.tsx

```diff
@@ -2,10 +2,12 @@ import { useEffect, useRef } from 'react'
 
 export const HtmlText = ({ str }: { str: string }) => {
   const ref = useRef<HTMLDivElement>(null)
+
   useEffect(() => {
     if (ref.current) {
       ref.current.innerHTML = str
     }
   }, [str])
+
   return <div ref={ref}></div>
```

Comment:
https://ckeditor.com/
If it necessary, please use dangerouslySetInnerHTML 
https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml

Reply:
Was used